guide/14-deep-learning/How CycleGAN Works.ipynb

```diff
@@ -0,0 +1,198 @@
+{
```

Comment:
<p>Change filename to 'how_cyclegan_works.ipynb' (no spaces, all lowercase)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>